test/WebJobs.Script.Tests/Description/FunctionInvokerBaseTests.cs

```diff
@@ -133,11 +133,14 @@ public void LogInvocationMetrics_EmitsExpectedEvents()
 
             Assert.Equal(2, completionEvents.Count);
             int invocation1Duration = (int.Parse(completionEvents[1].Groups["duration"].Value) / 100) * 100;
-            int invocation2Duration = (int.Parse(completionEvents[0].Groups["duration"].Value) / 100) * 100;
+
+            int invocation2RawDuration = int.Parse(completionEvents[0].Groups["duration"].Value); // save this for better output below.
+            int invocation2Duration = (invocation2RawDuration / 100) * 100;
 
             Assert.NotEqual(invocation1Duration, invocation2Duration);
             Assert.Equal(2000, invocation1Duration);
-            Assert.Equal(500, invocation2Duration);
+
+            Assert.True(invocation2Duration == 500, $"Expected 500. Actual: {invocation2Duration}. Raw value: {invocation2RawDuration}");
```

Comment:
Note: I've been seeing this fail quite often so I added some additional logging. This is unrelated to the change.